### PR TITLE
tools: mkimage: update to 2020.04

### PR DESCRIPTION
**Pull request by lucize**
also change the download source to https

Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

also dropped merged patches

### CODE REVIEW

**Comment by diizzyy** [comment]
I think you missed including the patches?

As for the list, don't drop the ftp.

Perhaps something like this would be a better option?
```
https://mirror.cyberbits.eu/u-boot
https://ftp.denx.de/pub/u-boot
ftp://ftp.denx.de/pub/u-boot
```
sources can be dropped as it's already used a last resort.

**Comment by lucize** [comment]
> https://mirror.cyberbits.eu/u-boot

for me I get 404 Not Found
the https and ftp server should be the same, does it matter ? usually https is accessed easyer from some restricted networks

File: tools/mkimage/Makefile [hunk 1]
```diff
@@ -7,11 +7,13 @@
...
 
 PKG_NAME:=mkimage
-PKG_VERSION:=2019.07
+PKG_VERSION:=2020.04
 
 PKG_SOURCE:=u-boot-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=http://ftp.denx.de/pub/u-boot
-PKG_HASH:=bff4fa77e8da17521c030ca4c5b947a056c1b1be4d3e6ee8637020b8d50251d0
+PKG_SOURCE_URL:= \
+	https://mirror.cyberbits.eu/u-boot\
```
Comment (diizzyy) [inline tools/mkimage/Makefile:14]: Missing space

**Comment by diizzyy** [comment]
Falling back to another protocol isn't necessarily a bad thing

**Comment by hauke** [comment]
Thank you for your patch, it was applied to master with some modifications.

**Pull request closed without merging**